geomesa-utils/src/main/scala/org/locationtech/geomesa/utils/geohash/GeohashUtils.scala

```diff
@@ -864,6 +864,18 @@ object GeohashUtils
     } yield newStr).take(maxSize + 1)
   }
 
+  def promoteToRegion(geom: Geometry): Geometry = geom match {
+    case g: Point   =>
+      g.buffer(1e-6)
```

Comment:
As an *insane* corner case, what happens if the Point to 'contain' is at one of the 4 'corners' of the world?

Just a quick thought experiment.  Probably worth merging...